From 02111237882170e0eae64a92c6bbb3261fe4d6b6 Mon Sep 17 00:00:00 2001
From: Daniel Zohm <daniel.zohm@aoemedia.de>
Date: Wed, 21 Nov 2012 17:07:20 +0100
Subject: [PATCH 6/6] silenced including class file

---
 lib/Varien/Autoload.php |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/Varien/Autoload.php b/lib/Varien/Autoload.php
index a0f266b..0a1a065 100644
--- a/lib/Varien/Autoload.php
+++ b/lib/Varien/Autoload.php
@@ -90,7 +90,7 @@ class Varien_Autoload
         }
         $classFile.= '.php';
         //echo $classFile;die();
-        return include $classFile;
+        return @include $classFile;
     }
 
     /**
-- 
1.7.0.4

